protocol/generated.proto

```diff
@@ -12,7 +12,9 @@ option go_package = "protocol";
 message ParseUASTRequest {
 	option (gogoproto.goproto_getters) = false;
 	option (gogoproto.typedecl) = false;
-	string content = 1;
+	string filename = 1;
+	string language = 2;
```

Comment:
It should not be included in the language field, otherwise it breaks a lot of our assumptions about language name being the same in the protocol, simple-linguist and driver image names.
We might have to add a separate language version field though.